core/src/main/resources/jenkins/security/ApiTokenProperty/config.properties

```
@@ -1,19 +1,17 @@
 TokenDisplayedOnce=Copy this token now, because it cannot be recovered in the future.
 AddNewToken=Add new Token
```

would make more sense for this to be lower case:
```suggestion
AddNewToken=Add new token
```